riscv/tests/riscv_data/keccak/src/main.rs

```diff
@@ -2,24 +2,34 @@
 #![no_std]
 
 extern crate powdr_riscv_runtime;
-use tiny_keccak::{Hasher, Keccak};
+use powdr_riscv_runtime::hash::Keccak;
```

Comment:
please leave this test as is so we still test with tinykeccak via software, and just make a copy of the test that uses the syscall instead.

Reply:
I end up putting the new test in `keccak_powdr`, which we had said was no longer useful

Reply:
Makes sense